src/app/loadout/Loadouts.tsx

```diff
@@ -392,6 +398,71 @@ function ItemCategory({
   );
 }
 
+function LoadoutParametersDisplay({ params }: { params: LoadoutParameters }) {
+  const defs = useD2Definitions()!;
+  const { query, exoticArmorHash, upgradeSpendTier, statConstraints, lockItemEnergyType } = params;
+  const show =
+    params.query ||
+    params.exoticArmorHash ||
+    params.upgradeSpendTier !== undefined ||
+    params.statConstraints?.some((s) => s.maxTier !== undefined || s.minTier !== undefined);
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <div className={styles.loParams}>
+      {query && (
+        <div className={styles.loQuery}>
+          <AppIcon icon={searchIcon} />
+          {query}
+        </div>
+      )}
+      {exoticArmorHash && (
+        <div className={styles.loExotic}>
+          <ExoticArmorChoice lockedExoticHash={exoticArmorHash} />
+        </div>
+      )}
+      {upgradeSpendTier !== undefined && (
+        <div className={styles.loSpendTier}>
+          <SelectedArmorUpgrade
+            defs={defs}
+            upgradeSpendTier={upgradeSpendTier}
+            lockItemEnergyType={lockItemEnergyType ?? false}
+          />
+        </div>
+      )}
+      {statConstraints && (
+        <div className={styles.loStats}>
+          {statConstraints.map((s) => (
+            <div key={s.statHash} className={styles.loStat}>
+              <BungieImage src={defs.Stat.get(s.statHash).displayProperties.icon} />
+              {s.minTier !== undefined && s.minTier !== 0 ? (
+                <span>
+                  {t('LoadoutBuilder.TierNumber', {
+                    tier: s.minTier,
+                  })}
+                  {(s.maxTier === 10 || s.maxTier === undefined) && s.minTier !== 10
+                    ? '+'
+                    : s.maxTier !== undefined && s.maxTier !== s.minTier
+                    ? `-${s.maxTier}`
+                    : ''}
+                </span>
+              ) : s.maxTier !== undefined ? (
+                <span>T{s.maxTier}-</span>
+              ) : (
+                t('LoadoutBuilder.TierNumber', {
+                  tier: 10,
+                }) + '-'
```

Comment:
I am hitting this case and I don't actually know what it means.

<img width="804" alt="Screen Shot 2021-11-21 at 10 50 25 pm" src="https://user-images.githubusercontent.com/7344652/142760690-2e9cf124-7695-4819-bf91-3f371cca8c9e.png">

Reply:
You will accept any tier, T10 or less

Reply:
I could also use `<=T10` if we thought that made more sense to people?

Reply:
To me accepting any is the same as no filters, so I would have expected nothing to be shown. Another option would be `T0-T10` all the time. 

I don't have an overly strong opinion but just wanted to call out that I didn't understand this when I initially saw it.

Reply:
Did you understand `T7+`?

We can't omit it when there's no stat range, since we're also using this to show the order and enabled-ness of stats.

Reply:
> Did you understand T7+?

Yeah I would understood that one, I am used to seeing a plus next to a number to indicate its that or greater. I haven't come across using minus for that or lower though.

> We can't omit it when there's no stat range, since we're also using this to show the order and enabled-ness of stats.

Yeah that is fair enough.